pages/linux/vcgencmd.md

```diff
@@ -0,0 +1,28 @@
+# vcgencmd
+
+> Print system information for a Raspberry Pi.
+> More information: <https://www.raspberrypi.org/documentation/raspbian/applications/vcgencmd.md>.
+
+- List all available commands:
+
+`vcgencmd commands`
+
+- Print the current CPU temperature:
+
+`vcgencmd measure_temp`
+
+- Print the current voltage usage:
+
+`vcgencmd measure_volts`
+
+- Print whether the CPU has throttled:
```

Comment:
I think this is more accurate, since it doesn't return true or false.
```suggestion
- Print the throttled state of the system as a bit pattern:
```